components/clp-package-utils/clp_package_utils/scripts/native/archive_manager.py

```diff
@@ -245,7 +250,9 @@ def _find_archives(
             db_conn.cursor(dictionary=True)
         ) as db_cursor:
             query_params: typing.List[int] = [begin_ts]
-            query: str = f"SELECT id FROM `{table_prefix}archives` WHERE begin_timestamp >= %s"
+            query: str = (
+                f"SELECT id FROM `{table_prefix}{ARCHIVES_TABLE_SUFFIX}` WHERE begin_timestamp >= %s"
```

Comment:
```suggestion
                f"""
                SELECT id FROM `{table_prefix}{ARCHIVES_TABLE_SUFFIX}`
                WHERE begin_timestamp >= %s
                """
```

To wrap to 100 chars.